XCUITests/TopTabsTest.swift

```diff
@@ -487,7 +491,9 @@ class TopTabsTestIpad: IpadOnlyTestCase {
         waitForValueContains(app.textFields["url"], value: url)
     }
 
-    func testTopSitesScrollToVisible() {
+    func testTopSitesScrollToVisible() throws {
```

Comment:
@SimonBasca I was trying to figure out the purpose of this test... I'm not sure... do you know? I have disabled it since I'm afraid it requires a major refactor...but please let me know if that's not the case.. I will open an issue to re-enable if that sounds fine...